sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -136,9 +134,11 @@ export interface FormSelectionMark extends FormElementCommon {
    */
   kind: "selectionMark";
   /**
-   * The state of the mark, either "selected" or "unselected".
+   * The state of the mark, either of:
+   * - "selected"
+   * - "unselected"
    */
-  state: SelectionMarkState;
+  state: "selected" | "unselected";
```

Comment:
curious: is this for a checkbox? could it be tri-state (indeterminate)?

Reply:
There was a point at which we thought it might eventually support indeterminate state, but after discussing with @xirzec, adding an indeterminate state would be a breaking change, so adding the strongest type makes sense here.